include/h2o/socket/evloop.h

```diff
@@ -53,6 +54,13 @@ typedef struct st_h2o_evloop_t {
     h2o_timerwheel_t *_timeouts;
     h2o_sliding_counter_t exec_time_nanosec_counter;
     uint64_t run_count;
+    struct {
+        uint64_t epoll_bp_usecs;
+        uint64_t epoll_bp_budget;
+        bool prefer_busy_poll;
```

Comment:
i wasn't sure where to put this comment, but i think that `prefer_busy_poll` should also have a configuration option like the other two options.

the use case for this would be:
  - regular busy polling (e.g. `mode: BUSYPOLL`)
  - the user wants the `gro-flush-timeout` to be deferrred (which is only possible if prefer_busy_poll is enabled)

Reply:
or to say it a bit differently:

- `prefer_busy_poll` can optionally be used with `mode: BUSYPOLL`, but is not required.
- `prefer_busy_poll` must be set to 1 for `mode: SUSPEND` to be enabled

Reply:
thank you, addressed in eae4a0a38cfeab4db52a2ddde07f2d0851acbaac